requirements.json

```diff
@@ -15,7 +15,8 @@
     "sha256sum": "59560c89b76f57ee72d9bd20bac047146f6f369fb42fba94ebaccd65c71209f5",
     "version": "11.0.0.GA",
     "useDownload": true,
-    "requires": ["jdk"]
+    "requires": ["jdk"],
+    "size": "645.38 MB"
```

Comment:
We should agree on units we use in size field (lets say Kilobytes) and put only number into this field. That would simplify total download computation. On UI side I would use angular filter and transform the number with humanize module to readable value.

Filter would look like:

``` JavaScript
module.filter('humanReadableSize', function() {
    return function(input) {
        return humanize.filesize(input*1000); //be careful with with kilobyte (x1000) and Kibibyte (x1024).
    }
});
```

then on template side:

``` html
{{ item.size | humanReadableSize }}
```